src/index.js

```diff
@@ -0,0 +1,19 @@
+import './style.css';
+import { handleRemoveTask, handleAddTask } from './modules/addRemoveTask.js';
+import { handleEditTask, handleWindowLoad } from './modules/renderEditTask.js';
+import {
+  handletaskStatus,
+  handleTaskDefaultStatus,
+} from './modules/taskStatus.js';
+import handleClearTask from './modules/clearTasks.js';
```

Comment:
I really like the way you laid out your code. it looks clean and simple is the best practice although I have a small optional suggestion i would like to recommend to you Maybe if you have a single function you may use only default export I'm sure that way it will look nice without so many curly braces. otherwise, everything looks great Job well Done.